Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
```

Comment:
Where are we using asciidoctor?

Reply:
we can have index.md or index.adoc :) same question was answered in november :)

Reply:
Yes, I remember briefly but I was under the impression that we were going to remove it.

Can you please convert 1 page of content as .asciidoc to make sure that this works as expected? I also think we should add a pointer to asciidoc documentation if the WG is going to support this on the site now.

FWIW - My preference would be to create a separate issue/pr for this since this can change how people contribute content to the whole site, not just news/announcement.

Reply:
To clarify any confusion, I am 100% on board with adding .adoc support!

Reply:
I originally had few examples in .adoc, but it was lorem ipsum :) will look into conversion do you want it in this PR?

Reply:
I don't believe it's urgent to show an example, this PR is in the pipes from September, I would prefer doing additional stuff in separate PR.